docs/source/containers/singularity.md

```diff
@@ -0,0 +1,101 @@
+(singularity)=
+# Singularity on Discovery
+
+Singularity is the container runtime engine for the Discovery cluster and is installed as a module. With Singularity, you can run existing containers or build and run your own custom container.
```

Comment:
I would remove the comment on it can be used to build containers. Maybe pull containers?